src/components/BookList.js

```diff
@@ -1,26 +1,18 @@
 import React from 'react';
-import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+
 import Book from './Book';
 
-function BookList({ books, onDelete }) {
+function BookList() {
```

Comment:
- [ ] You have an error in the browser console saying that you should make sure each of the list items have a key prop. Please fix that
![image](https://github.com/cancelei/bookstore/assets/45460192/eedeba92-8eb3-4df0-a0ab-800d9b382949)